arvo/jael/tasks.md

```diff
@@ -0,0 +1,238 @@
+
+## Returns
+
+Jael does not return any `gift`s in response to a `%moon` `task`.
+
+# `%nuke`
+
+```hoon
+[%nuke whos=(set ship)]
+```
+
+Cancel subscription to public or private key updates.
+
+If you've subscribed to public or private key updates from Jael with a [%private-keys](#private-keys) or [%public-keys](#public-keys) `task`, you can unsubscribe and stop receiving updates with a `%nuke` `task`. The `(set ship)` is the `set` of `ship`s which you want to stop tracking. Jael organises subscriptions based on `duct`s, and will determine which subscription to cancel implicitly based on the `duct` the `%nuke` `task` came from. This means a `%nuke` `task` only works from the same thread or agent and on the same `path` as the original subscription request.
+
+To cancel a subscription to a ship's private keys you must leave `whos` empty like `[%nuke ~]`.
+
+## Returns
+
+Jael does not return a `gift` in response to a `%nuke` `task`.
+
+## Examples
+
+See the [%public-keys and %nuke](@/docs/arvo/jael/examples.md#public-keys-and-nuke) section of the Examples document for an example of using `%nuke` to cancel a `%public-keys` subscription. See the thread in the [%private-keys](@/docs/arvo/jael/examples.md#private-keys) example for cancelling a `%private-keys` subscription.
+
+# `%private-keys`
+
+```hoon
+[%private-keys ~]
+```
+
+Subscribe to private key updates.
+
+An agent or thread can subscribe to be notified of private key updates for the local ship. The subscription will continue until Jael receives a [%nuke](#nuke) `task` to cancel it.
+
+## Returns
+
+Jael responds to a `%private-keys` `task` with a `%private-keys` `gift` which looks like:
+
+```hoon
+[%private-keys =life vein=(map life ring)]
+```
+
+The `life` is the current life of the ship and the `vein` `map` contains the private key for each life up to (and including) the current one. Upon first subscribing, Jael will immediately respond with a `%private-keys` `gift`. Then, whenever the ship's private keys are changed, it'll send a new and updated `%private-keys` `gift`.
+
+## Example
+
+See the [%private-keys](@/docs/arvo/jael/examples.md#private-keys) section of the Examples document for a practical example.
+
+# `%public-keys`
+
+```hoon
+[%public-keys ships=(set ship)]
+```
+
+Subscribe to public key (and related) updates from Jael.
+
+An agent or thread can subscribe to be notified of public key updates, sponsorship changes and continuity breaches for the `set` of `ship`s specified in the `ships` field. The subscription will continue until Jael receives a [%nuke](#nuke) `task` to cancel it.
```

Comment:
> An agent or thread can ...

Not sure why these are mentioned explicitly. Technically any part of the system that can emit moves to arvo can do this. Ames uses this, for example.